### ISSUE TITLE
input "cache" is not defined in action "actions/setup-python@v2". available inputs are "architecture", "python-version", "token"

### ISSUE THREAD

**Issue by toast-gear** [rhysd/actionlint#95]
**Error** 
```
input "cache" is not defined in action "actions/setup-python@v2". available inputs
are "architecture", "python-version", "token"
```

**Steps to Reproduce**

1. Create this workflow:

```yaml
name: My Workflow

on:
  workflow_dispatch:

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Set up python
        uses: actions/setup-python@v2
        with:
          python-version: 3.10
          cache: 'pip'
```

2. Lint this workflow with actionlint

**Links**
https://github.com/actions/setup-python#caching-packages-dependencies caching is a somewhat new input for this action
https://github.com/actions/setup-python/blob/main/action.yml there are a few additional inputs compared to the current whitelist

**Comment by xsc27** [comment]
@toast-gear, thank you for reporting this, as I have the same issue. And I have a similar problem w/ another action that has added inputs. Linting appears to pass if a sha is used instead of a tag.

**Comment by StevenMaude** [comment]
For `setup-python@v2`, at least, this is fixed if you build [current `main`](https://github.com/rhysd/actionlint/blob/0c923eac01a80b5a0b417f0240164c8806753203/popular_actions.go#L625-L636) (or any commit since f21cdb8c37e4b2fda20648faba85c0537de1fa07).

With that change, you no longer need to use a SHA hash in place of `v2` in `setup-python@v2` to get actionlint to pass for the use of features like `cache`. It's just that the [last GitHub release is from over three months ago](https://github.com/rhysd/actionlint/releases/tag/v1.6.8) and doesn't have the updated actions information.

**Comment by rhysd** [comment]
This will be fixed soon. I'm sorry for inconvenience.

**Comment by rhysd** [comment]
This was fixed by v1.6.9

**Comment by StevenMaude** [comment]
@rhysd: Thanks for the new release with this fix! :rocket::heart:
